frontend/src/scenes/billing/v2/PlanTable.tsx

```diff
@@ -1,408 +1,163 @@
-                name: 'Advanced permissions',
-                value: true,
-            },
-            sso: {
-                name: 'SAML SSO',
-                value: true,
-            },
-        },
-        products: [
-            {
-                name: 'Product analytics',
-                tiers: [
-                    {
-                        description: 'First 1 million events/mo',
-                        price: 'Included',
-                    },
-                    {
-                        description: '1-2 million',
-                        price: '$0.0005625',
-                    },
-                ],
-            },
-            {
-                name: 'Session recording',
-                tiers: [
-                    {
-                        description: 'First 15,000 recordings/mo',
-                        price: 'Included',
-                    },
-                    {
-                        description: '15,000-50,000',
-                        price: '$0.00625',
-                    },
-                ],
-            },
-        ],
-    },
-]
-
 export function PlanIcon({
-    value,
-    note,
+    feature,
     className,
+    timeDenominator,
 }: {
-    value: boolean | string
-    note?: string
+    feature?: BillingV2FeatureType
     className?: string
+    timeDenominator?: string
 }): JSX.Element {
     return (
         <div className="flex items-center text-xs text-muted">
-            {value === true ? (
+            {!feature ? (
                 <>
-                    <IconCheckmark className={`text-success mr-4 ${className}`} />
-                    {note}
+                    <IconClose className={`text-danger mr-4 ${className}`} />
                 </>
-            ) : value === false ? (
+            ) : feature.limit ? (
                 <>
-                    <IconClose className={`text-danger mr-4 ${className}`} />
-                    {note}
+                    <IconWarning className={`text-warning mr-4 ${className}`} />
+                    {feature.limit &&
+                        `${convertLargeNumberToWords(feature.limit, null)} ${feature.unit && feature.unit}${
+                            timeDenominator ? `/${timeDenominator}` : ''
+                        }`}
+                    {feature.note}
                 </>
             ) : (
                 <>
-                    <IconWarning className={`text-warning mr-4 ${className}`} />
-                    {note}
+                    <IconCheckmark className={`text-success mr-4 ${className}`} />
+                    {feature.note}
                 </>
             )}
         </div>
     )
 }
 
+const getPlanBasePrice = (plan: BillingV2PlanType): number | string => {
+    const basePlan = plan.products.find((product) => product.type === 'enterprise' || product.type === 'base')
+    if (basePlan?.unit_amount_usd) {
+        return `$${parseInt(basePlan.unit_amount_usd)}/mo`
+    }
+    if (plan.name === 'Starter') {
```

Comment:
This is too hardcoded for the scope of what we're doing :D

Reply:
Oops, will update.